### ISSUE TITLE
Use intent filters to add new dokuwikis

### ISSUE THREAD

**Issue by timroes** [DokuWikiMobile/dokuwikimobile#32]
A new schema "dokuwiki://" should be introduced, to add new dokuwikis. A link to that protocol could be generated by a to-develop dokuwiikiplugin.

Also an intent filter for "/lib/exe/xmlrpc.php" could be made, since the QR codes should contain a http/https link (not a custom dokuwiki:// link), since the most used barcode scanner doesn't offer an "open" button for non http/https links. So scanning the QR code directly from the barcode scanner wouldn't invoke the DokuWiki app.

In short:
- "dokuwiki" use for links (e.g. in browser, so the browser won't open them, but the dokuwiki app gets invoked)
- "http/https" for QR code, so the barcode scanner displays an open button to the user.
